Source/PartnerCenter.CustomerPortal/BusinessLogic/Commerce/PaymentGateways/PayU/PostBackParam.cs

```diff
@@ -0,0 +1,369 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="PostBackParam.cs" company="Microsoft">
+//      Copyright (c) Microsoft Corporation.  All rights reserved.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Microsoft.Store.PartnerCenter.CustomerPortal.BusinessLogic.Commerce.PaymentGateways
+{
+    /// <summary>
+    /// PostBack Parameter class
+    /// </summary>
+    public class PostBackParam
+    {
+        /// <summary>
+        /// Gets or sets added on
+        /// </summary>
+        public string Addedon { get; set; }
```

Comment:
Adding to @tameemansari comments can we rename all the properties with underscores as well? Instead of Card_merchant_param something like CardMerchantParam as an example